File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        for(int i=1;i<nums.length;i++){
            for(int j=0;j<i;j++){
                if(nums[j]== nums[i]){
                    return true;
                }
            }
        }
        return false;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.cpp
class Solution {
public:
    int longestConsecutive(vector<int>& nums) {
        unordered_set<int> store(nums.begin(), nums.end());

        int res=0, curr=0, streak=0;

        for(int num:nums){
            curr= num;
            while(store.find(curr)!= store.end()){

                curr++;
                streak++;
            }
            res= max(res, streak);
            streak=0;
        }

        return res;
    }
};
